files/default/pgrestore-ckan.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+set -x
+
+NOW=$(date +%s)
+BACKUP_DIR=/data/db_backups
```

Comment:
Please add a 'are you sure' or in 'dryrun' mode before running for real.

Reply:
Done.